blocks/stats/stats.css

```diff
@@ -0,0 +1,81 @@
+.statsWrapper {
+    background-color: var(--c-dark);
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin: 0 auto;
+}
+
+.stats.block {
+    display: flex;
+    place-content: center;
+}
+
+.stat-title {
+    align-items: center;
+    color: var(--c-white);
+    display: flex;
+    font-family: var(--ff-primary);
+    font-size: 50px;
+    font-weight: 700;
+    height: auto;
+    line-height: 56px;
+    padding: 16px;
+    text-align: center;
+}
+
+.line {
+    background: linear-gradient(135deg, var(--c-secondary-pink) 0%, var(--c-primary-lt) 51.1%, var(--c-secondary-orange-lt) 100%);
+    height: 5px;
+    width: 151px;
+}
+
+.stat {
+    align-items: center;
+    display: flex;
+    flex-basis: 100%;
+    flex-direction: column;
+    padding: 39px 0;
+}
+
+.stat-description {
+    color: var(--c-white);
+    font-family: var(--ff-secondary);
+    font-size: 18px;
+    font-weight: 700;
+    line-height: normal;
+    max-width: 400px; 
+    padding: 16px;
+    text-align: center;
+}
+
+.noBackground {
+    background-color: var(--c-white);
+}
+
+.noBackground .stat-title,
+.noBackground .stat-description {
+    color: var(--c-copy);
+}
+
+.noBackground .stat-description {
+    font-weight: 400;
+    line-height: 24px;
+}
+
+.stat-title.large {
+    font-size: 70px;
+}
+
+sup {
```

Comment:
Don't use the "sup" selector directly. It will affect everywhere in the site. Please use the component selector like this ".stats sup"